### PR TITLE
chore: prepare testnet 75

### PR DESCRIPTION
**Pull request by conorsch**
## Describe your changes

Makes three substantive changes:

  * resets `TOTAL_HALT_COUNT` to 0
  * increments `APP_VERSION`
  * increments chain_id `penumbra-testnet-deimos-7` -> `penumbra-testnet-deimos-8`

There are multiple chain id directories included, pulled from the `release/v0.73.2` tag. Due to our multi-branch workflow, we never backported those testnet configs to the main branch, but doing so is necessary now that we're releasing from main again.

Refs #4374.

## Checklist before requesting a review

- [x] If this code contains consensus-breaking changes, I have added the "consensus-breaking" label. Otherwise, I declare my belief that there are not consensus-breaking changes, for the following reason:

  > This PR is in prep for a full chain reset.

### CODE REVIEW

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 | 1 (Testnet 70)            | v0.70.x                | v0.37.5  |   v1     |
 | 2 (Testnet 73)            | v0.73.x                | v0.37.5  |   v1     |
 | 3 (Testnet 74)            | v0.74.x                | v0.37.5  |   v1     |
+| 4 (Testnet 75)            | v0.75.x                | v0.37.5  |   v1     |
```
Comment (conorsch) [inline COMPATIBILITY.md:6]: Bumping crate version only in the COMPAT doc for now. Post-merge, we'll bump-and-tag directly on the main branch, to trigger the release.

**Review by cratelyn — APPROVED** [review]
> There are multiple chain id directories included, pulled from the release/v0.73.2 tag. Due to our multi-branch workflow, we never backported those testnet configs to the main branch, but doing so is necessary now that we're releasing from main again.

thanks for explaining the backing context. this looks great.

**Comment by cratelyn** [comment]
one non-blocking question, since github review does not provide a nice way to hang comments on a parent directory:

for extra clarity, i might suggest that we leave e.g. an empty tombstone / stub directory, `testnets/074-deimos-7`. this could contain a small `README.md` containing a small memorandum about why that migration failed, noting that the jump from `73` to `75` was intentional and related to an operational issue?

**Comment by conorsch** [comment]
> for extra clarity, i might suggest that we leave e.g. an empty tombstone / stub directory, testnets/074-deimos-7. this could contain a small README.md containing a small memorandum about why that migration failed, noting that the jump from 73 to 75 was intentional and related to an operational issue.

This is a good suggestion! There are similar events would could backfill for posterity. The testnet dirs themselves are machine-readable, and their contents are baked into binaries, so I'd prefer to keep READMEs out of band. But a file in `testnets/` recording the history is a gift to future maintainers, for sure.

**Comment by cratelyn** [comment]
> > for extra clarity, i might suggest that we leave e.g. an empty tombstone / stub directory, testnets/074-deimos-7. this could contain a small README.md containing a small memorandum about why that migration failed, noting that the jump from 73 to 75 was intentional and related to an operational issue.
> 
> This is a good suggestion! There are similar events would could backfill for posterity. The testnet dirs themselves are machine-readable, and their contents are baked into binaries, so I'd prefer to keep READMEs out of band. But a file in `testnets/` recording the history is a gift to future maintainers, for sure.

ah-ha! that is very good to know, had not realized these directories are machine-readable. knowing that, i agree with that plan of action. thank you for elaborating!

**Pull request merged**